lib/main_common.pm

```diff
@@ -1487,6 +1487,7 @@ sub load_extra_tests_textmode {
     unless (is_sle '<15') {
         loadtest "console/ntp_client";
     }
+    loadtest "console/mta";
```

Comment:
As we enable it for all distribution, could you please provide verification runs for SLE 15 and openSUSE?